src/Microsoft.Health.SqlServer/Features/Schema/Manager/SqlSchemaManager.cs

```diff
@@ -119,7 +119,18 @@ public virtual async Task ApplySchema(MutuallyExclusiveType type, bool force = f
                 throw new SchemaManagerException(string.Format(CultureInfo.CurrentCulture, Resources.SpecifiedVersionNotAvailable, targetVersion));
             }
 
-            await ValidateVersionCompatibility(availableVersions[^1].Id, token).ConfigureAwait(false);
+            // TTL in instance schema table can cause this call to fail on first attempt so allow for retries
+            attemptCount = 1;
+            await Policy.Handle<SchemaManagerException>()
+            .WaitAndRetryAsync(
+                retryCount: RetryAttempts,
+                sleepDurationProvider: (retryCount) => TimeSpan.FromSeconds(60),
+                onRetry: (exception, retryCount) =>
+                {
```

Comment:
nit: use expression over block (which I guess is a type of expression) for single line delegates